SensusService/DataStores/DataStore.cs

```diff
@@ -378,26 +404,23 @@ public DataStore Copy()
                 PreserveReferencesHandling = PreserveReferencesHandling.None,
                 ReferenceLoopHandling = ReferenceLoopHandling.Ignore,
                 TypeNameHandling = TypeNameHandling.All,
-                ConstructorHandling = ConstructorHandling.AllowNonPublicDefaultConstructor
+                ConstructorHandling = ConstructorHandling.AllowNonPublicDefaultConstructor,
```

Comment:
Again, Wes's changes but maybe a little nitpicky in my opinion? The code compiles either way.